### PR TITLE
Homework Git added

### PR DESCRIPTION
**Pull request by DomenZero**
Hello!

Tasks 1 and 2 have a README description
Task 3 screenshots in the 3_gitlab.tar
Task 4* screenshot attached as TravisCI.jpg

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  (hw_git)$ git add test
+  (hw_git)$ git commit -m "Made: create file test"
+  ```
+  4. Switch back to master branch and add anything else to the empty file "homework" there too
+  ```git
+  (hw_git) $ echo "Hello" > homework
+  (hw_git) $ cat homework 
+  Hello
+  (hw_git) $ git add homework
+  warning: LF will be replaced by CRLF in homework.
```
Comment (SagurovA93) [inline README.md:36]: What does it mean and how to remove such warnings?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  ```
+  7. Revert to the first commit in "hw_git" branch
+  ```git
+  $ git log --pretty=oneline
+  $ git revert c5b3385e2221
+  OR
+  $ git revert HEAD~1
+  ```
+
+2.
```
Comment (SagurovA93) [inline README.md:58]: Such indentations won't work for MD files. You may use either '###' for new headers or more indentations below your block ([link](https://stackoverflow.com/questions/37575916/how-to-markdown-nested-list-items-in-bitbucket))

**Review by SagurovA93 — APPROVED** [review]
Good. 

Some additional questions: 
1. What will happen when your local GitLab docker container will stop? Will it keep configurations and code?
2. How to setup root password before Gitlabb starts?

File: README.md (hunk 1 above)
Comment (DomenZero) [reply README.md:36]: - What will happen when your local GitLab docker container will stop? Will it keep configurations and code?
If we don't have volumes then data may be lost along with stopping of container

- How to setup root password before Gitlab starts?
Maybe I could add GITLAB_ROOT_PASSWORD
For example
`sudo docker run --detach --hostname gitlab.epam_hw.com --publish 80:80 --name gitlab-epam_hw --restart always  --volume /srv/gitlab/config:/etc/gitlab --volume /srv/gitlab/logs:/var/log/gitlab --volume /srv/gitlab/data:/var/opt/gitlab --env GITLAB_ROOT_PASSWORD="Пароль" GITLAB_OMNIBUS_CONFIG="external_url 'http://192.168.0.121/';" gitlab/gitlab-ce:latest `

- What does it mean and how to remove such warnings?
I couldn't see this message if I will use Unix OS. 
But I use Windows and I can remove it using the next command "git config core.autocrlf true" which turn off this warning.
The warning means the distinction between at the end of a line with a carriage return (CR) and a line feed (LF) in Windows and only LF in Unix.

File: README.md (hunk 2 above)
Comment (DomenZero) [reply README.md:58]: Thanks for the help)

**Comment by SagurovA93** [comment]
Good, this PR could be merged.

**Pull request merged**